docs/asdf/config.rst

```diff
@@ -204,13 +204,13 @@ legacy_fill_schema_defaults
 ---------------------------
 
 Flag that controls filling default values from schemas for older versions of
-the ASDF Standard.  This library used to remove nodes from the tree whose
+the ASDF.  This library used to remove nodes from the tree whose
```

Comment:
Missing something?

Reply:
Updated in https://github.com/asdf-format/asdf/pull/1953/commits/b4327411b79c9582dbadd1c2fbcf65f975ddc351